README.md

```diff
@@ -51,6 +51,19 @@ add the following to `app/src/main/res/values/strings.xml`:
 <string name="app_update_install">RESTART</string>
 ```
 
+Configuration Preferences
+------------
+
+Non US-AppStore iOS apps
```

Comment:
Can we make this a subheading with `###`?

Reply:
👍